Makefile

```diff
@@ -3,3 +3,6 @@ build:
 
 test:
 	@sh scripts/test.sh
+
+local_setup:
```

Comment:
@Palleas tell me if you have a better name for this. I would like this directive to be a verb like the other ones

Reply:
I like "bootstrap" or "setup_env"

Reply:
I like `setup_env`